tests/job-spawner-and-cleaner.py

```diff
@@ -4,22 +4,28 @@
 
 import redis
 
+
 sys.path.append('../python')
 from redis_work_queue import KeyPrefix, Item, WorkQueue
 
+queue_list = sys.argv[2].split(" ")
+print(queue_list)
+
 if len(sys.argv) < 2:
     raise Exception("first command line argument must be redis host")
-db = redis.Redis(host=sys.argv[1])
+db = redis.Redis(host=sys.argv[1].replace(":6379",""))
```

Comment:
Why not split the argument into host and port and pass these both? With this, if I pass `localhost:4567` it won't connect to the right port!